assets/src/edit-story/components/library/text/fontPreview.js

```diff
@@ -68,7 +68,7 @@ function FontPreview({ title, element, onClick }) {
   }, [font, fontWeight, content, maybeEnqueueFontStyle]);
 
   return (
-    <Preview onClick={onClick}>
+    <Preview onClick={onClick} fontSize={fontSize}>
```

Comment:
Why is this here - the `fontSize` prop?

Reply:
Oh good catch. I was playing with something but forgot to remove. will take it out